### DISCUSSION TITLE
Sorting plugins

### DISCUSSION THREAD

**Discussion by gianfrancodemarco** [lowlighter/metrics#1473] [Q&A] [answered]
Hi all,
this is probably a very basic question but I couldn't find the answer.

How can I sort the plugins to give an order in the generated output?
E.g.: I want first the content from base, then the content from the topics plugin etc.

Thank you

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

Sorry for the late answer,
You can achieve this through [`config_order`](https://github.com/lowlighter/metrics/blob/master/source/plugins/core/README.md#config_order) option (See [`📦 Ordering content`](https://github.com/lowlighter/metrics/blob/master/source/plugins/core/README.md#-ordering-content) for more informations) 🙂

**Comment by gianfrancodemarco** [comment]
I missed that part of the docs, thank you!
